src/components/Advertising.tsx

```diff
@@ -35,29 +35,31 @@ const Advertising: React.FC = () => {
     <Flex
       flex="auto"
       flexDirection={["column", "column", "column", "row", "row"]}
+      justifyContent="center"
```

Comment:
Is this fine for both `row` and `column` layouts?

Reply:
I just removed the second to last `"row"` actually as this is unnecessary, but yeah it seems to be fine for both